hadoop-ozone/client/src/main/java/org/apache/hadoop/ozone/client/protocol/ClientProtocol.java

```diff
@@ -124,6 +124,7 @@ OzoneVolume getVolumeDetails(String volumeName)
    * @return Boolean - True if the user with a role can access the volume.
    * This is possible for owners of the volume and admin users
    * @throws IOException
+   * TODO: deprecate and remove this API.
```

Comment:
Shall we add a @Deprecated annotation on this API?

Reply:
Added.